### DISCUSSION TITLE
Homebrew portaudio fails on BigSur

### DISCUSSION THREAD

**Discussion by dl1ycf** [Homebrew/discussions#526] [Everyday usage]
Perhaps a duplicate: when using portaudio with homebrew on MacOS 10.11 "Big Sur", portaudio applications fail unless you link with the "old" SDK. After upgrading, you have to
- delete XCode app
- delete everything in /Library/Developer
- obtain and install XCode 12.1 from Apple developer site, and the 12.1 command line tools
- DO NOT upgrade XCode or the command line tools
After this, you can create portaudio applications under MacOS 10.11.

Reason: portaudio uses deprecated functions to maintain compatibility with very old MacOS versions, and this no longer works with the latest SDK. The MacPorts people have made a correction that is already integrated upstream but not fully.

**Comment by carlocab** [comment]
Isn't macOS 10.11 El Capitan?

**Comment by dl1ycf** [comment]
obviously a typo, the OS version is not 10.11 but 11.1

**Comment by dl1ycf** [comment]
But can you confirm that there is problem with portaudio after
updating to BigSur? And if the problem arises, can you confirm
that installing Xcode 12.1 from Apple’s archive solves the problem?

This is what I found out, but is it generally true or some artifact
in my installation?
